src/main/java/seedu/storemando/logic/commands/ListCommand.java

```diff
@@ -17,13 +20,25 @@
     public static final String COMMAND_WORD = "list";
 
     public static final String MESSAGE_SUCCESS = "Listed all items";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": List items in the storemando. "
+        + "Parameters: "
+        + "[" + PREFIX_LOCATION + "LOCATION] "
```

Comment:
Would this message usage be misleading and give users the idea that both tag and location can be specified? Would it be better if the message usage is [l/LOCATION]**/**[t/TAG]?